modules/ossm-kiali-ossmc-plugin-install-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-kiali-ossmc.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-kiali-ossmc-install-web-cli_{context}"]
+= Installing OpenShift Service Mesh Console plugin using CLI
+
+You can install the OpenShift Service Mesh Console (OSSMC) plugin using the CLI instead of the OpenShift Console.
+
+.Prerequesites
+
+* OpenShift v4.11+
+* Istio with Kiali
```

Comment:
I don't know what that means "Istio with Kiali" because the 4th bullet point mentioned "Kiali Server".
Perhaps just remove "with Kial" here.

However you want to say it... one prereq is "Istio" needs to be installed and a second prereq is "Kiali" needs to be installed.